### ISSUE TITLE
decapoda-research/llama-7b-hf no longer accessible on Huggingface

### ISSUE THREAD

**Issue by coree** [TrustedLLM/LLMDet#6]
The [decapoda-research/llama-7b-hf](https://huggingface.co/decapoda-research/llama-7b-hf) model, which is used in the [detector.py](https://github.com/TrustedLLM/LLMDet/blob/2c16aaf2b1cc8102beadd92343d64af51331e6e4/llmdet/detector.py#L114) file, is no longer accessible on HuggingFace. Can you tell me if the [decapoda-research/llama-7b-hf](https://huggingface.co/decapoda-research/llama-7b-hf) model can be replaced by the accessible model [baffo32/decapoda-research-llama-7B-hf](https://huggingface.co/baffo32/decapoda-research-llama-7B-hf) ?

**Comment by cansee5** [comment]
Yes, you can replace [decapoda-research/llama-7b-hf](https://huggingface.co/decapoda-research/llama-7b-hf) with [baffo32/decapoda-research-llama-7B-hf](https://huggingface.co/baffo32/decapoda-research-llama-7B-hf).
